solitude/base.py

```diff
@@ -719,4 +738,14 @@ class NonDeleteModelViewSet(
     Same as the ModelViewSet, without the DeleteMixin. Uses our local mixins
     to give us ETag support.
     """
-    pass
+
+    # Allow tastypie resources to connect to viewsets, to delete
+    # once tastypie is gone. You really don't want to know.
+    def get_via_uri(self, uri, request=None):
```

Comment:
Hmm haven't got that working without these hacks. The good thing is these are all temporary and I hope to remove them by the end of the week.